File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0803U00grdOCS0803ItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
    public class OCS0803U00grdOCS0803ItemInfo
    {
        private String _patStatusGr;
        private String _patStatusGrName;
        private String _ment;
        private String _seq;
        private String _rowState;

        public String PatStatusGr
        {
            get { return this._patStatusGr; }
            set { this._patStatusGr = value; }
        }

        public String PatStatusGrName
        {
            get { return this._patStatusGrName; }
            set { this._patStatusGrName = value; }
        }

        public String Ment
        {
            get { return this._ment; }
            set { this._ment = value; }
        }

        public String Seq
        {
            get { return this._seq; }
            set { this._seq = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public OCS0803U00grdOCS0803ItemInfo() { }

        public OCS0803U00grdOCS0803ItemInfo(String patStatusGr, String patStatusGrName, String ment, String seq, String rowState)
        {
            this._patStatusGr = patStatusGr;
            this._patStatusGrName = patStatusGrName;
            this._ment = ment;
            this._seq = seq;
            this._rowState = rowState;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0803U00grdOCS0803ItemInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCS0803U00grdOCS0803ItemInfo {
	private String patStatusGr   ; 
	private String patStatusGrName;
	private String ment ;
	private Double seq;
	public OCS0803U00grdOCS0803ItemInfo(String patStatusGr,
			String patStatusGrName, String ment, Double seq) {
		super();
		this.patStatusGr = patStatusGr;
		this.patStatusGrName = patStatusGrName;
		this.ment = ment;
		this.seq = seq;
	}
	public String getPatStatusGr() {
		return patStatusGr;
	}
	public void setPatStatusGr(String patStatusGr) {
		this.patStatusGr = patStatusGr;
	}
	public String getPatStatusGrName() {
		return patStatusGrName;
	}
	public void setPatStatusGrName(String patStatusGrName) {
		this.patStatusGrName = patStatusGrName;
	}
	public String getMent() {
		return ment;
	}
	public void setMent(String ment) {
		this.ment = ment;
	}
	public Double getSeq() {
		return seq;
	}
	public void setSeq(Double seq) {
		this.seq = seq;
	}
	

}
